puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_develop.yaml

```diff
@@ -0,0 +1,49 @@
+- job:
+    name: test_develop
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 32
+    properties:
+      - github_foreman
```

Comment:
Might as well put this inline, it's only really going to be applicable for this.